docs/maintain/maintain-guides-how-to-validate-polkadot.md

```diff
@@ -572,13 +572,15 @@ other peers over the network.
 ## VPS List
 
 - [OVH](https://www.ovh.com.au/)
-- [Digital Ocean](https://www.digitalocean.com/)
+- [Digital Ocean](https://www.digitalocean.com/)*
 - [Vultr](https://www.vultr.com/)
 - [Linode](https://www.linode.com/)
 - [Contabo](https://contabo.com/)
 - [Scaleway](https://www.scaleway.com/)
 - [OnFinality](https://onfinality.io/)
 
+*Beware that Digital Ocean lists "Mining of Cryptocurrencies" under the Network Abuse section of their [Acceptable Use Policy](https://www.digitalocean.com/legal/acceptable-use-policy/), and if your account is considered to be in conflict with that, it may be locked and your server shut down.
```

Comment:
```suggestion
 > Beware of the **Terms and Conditions** and **Acceptable Use Policies** for each VPS provider. 
 You may be locked out of your account and your server shut down if you come in violation.
 For instance, Digital Ocean lists "Mining of Cryptocurrencies" under the Network Abuse section of
 their [Acceptable Use Policy](https://www.digitalocean.com/legal/acceptable-use-policy/) and requires
 explicit permission to do so. This may extend to other cryptocurrency activity.
```

Reply:
We can word it as a general statement and use Digital Ocean as an example since the list of providers is meant to serve as options. The choice, and the responsibility of that choice, is ultimately on the user.